.github/workflows/build-deb-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Test Deb Package
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  pull_request:
+jobs:
+  build-test-deb-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: chmod 755 homeassistant-supervised/DEBIAN/p*
```

Comment:
Why is this needed?
Is the git index not correct?
755 seems overkill?

Reply:
I have set it to 555 now